src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -134,13 +140,30 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
     };
   }, [relatedVenueEvents, userEventIds, selectedDayIndex, getEventLocation]);
 
+  const downloadCalenderPersonal = useCallback(() => {
+    const allPersonalEvents: PersonalizedVenueEvent[] = relatedVenueEvents
+      .map(prepareForCalendar(userEventIds))
+      .filter((event) => event.isSaved);
+    createCalendar(allPersonalEvents, "Personal");
+  }, [relatedVenueEvents, userEventIds]);
```

Comment:
Whitespace makes code easier to read.

```suggestion
  const downloadCalenderPersonal = useCallback(() => {
    const allPersonalEvents: PersonalizedVenueEvent[] = relatedVenueEvents
      .map(prepareForCalendar(userEventIds))
      .filter((event) => event.isSaved);

    createCalendar(allPersonalEvents, "Personal");
  }, [relatedVenueEvents, userEventIds]);
```